iroh-cli/src/config.rs

```diff
@@ -10,7 +10,7 @@ use std::{
 
 use anyhow::{anyhow, bail, Context, Result};
 use iroh::net::{
-    defaults,
+    defaults::{self},
```

Comment:
this looks, odd

Reply:
yea, weird. auto import or something. I'll fix it.